screen-settings/src/main/java/com/ivy/settings/SettingsScreen.kt

```diff
@@ -625,9 +627,18 @@ private fun ContactSupport() {
 }
 
 @Composable
-private fun Contributors() {
-    val nav = navigation()
+private fun Releases(nav: Navigation) {
+    SettingsDefaultButton(
+        icon = R.drawable.ic_vue_money_tag,
+        text = "Releases",
+        iconPadding = 6.dp
```

Comment:
Increase the icon padding to let's say to `12.dp` so the icon doesn't look enormous